src/frontend/screens/Onboarding/walletSelection/screens/import/index.tsx

```diff
@@ -61,26 +65,30 @@ const ImportScreen = ({
           `By importing, your MetaMask installation and  settings will be imported into HyperPlay.`
         )}
       </div>
-      {browserSelected === '' ? (
+      {browserSelected === null ||
+      importOptions[browserSelected] === undefined ? (
         <div className={ImportScreenStyles.importOptionsContainer}>
-          {importOptions &&
-            Object.keys(importOptions).map((browser) => {
-              return (
-                <ImportOption
-                  onClick={async () => setBrowserSelected(browser)}
-                  title={browser as ImportableBrowser}
-                  key={browser}
-                />
-              )
-            })}
+          {importOptions !== undefined
```

Comment:
this one also looks convoluted, could be replaced by a helped function imo.

Reply:
sure thing. cleaned it up